src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndProjData.cxx

```diff
@@ -729,15 +732,15 @@ actual_compute_subset_gradient_without_penalty(TargetT& gradient,
                                  this->proj_data_sptr, 
                                  subset_num, 
                                  this->num_subsets, 
-                                 -this->max_segment_num_to_process,
+                                 0,
```

Comment:
This seems the only non-TOF change. A mistake?

Reply:
looks like a mistake